src/libs/icon-dict.js

```diff
@@ -38,40 +38,40 @@ const iconDict = {
   'angle up': custom(angleUp),
   'angle-double left': rotate(-90, custom(angleDoubleUp)),
   'angle-double right': rotate(90, custom(angleDoubleUp)),
-  'ban': fa(faBan),
-  'bars': custom(bars),
-  'check': fa(faCheck),
-  'clock': fa(faClock),
-  'cloud': fa(faCloud),
-  'cog': fa(faCog),
-  'copy': fa(faClone),
+  ban: fa(faBan),
```

Comment:
Is this our code or something we've copied in from somewhere else? If it's ours, why would we choose `'angle down'` as a property name? If it's not ours, should we make an exception to the quote props rule?

Reply:
This is our code. I am not sure what the rationale is behind having a space in the prop name. I'll change these names and chat with Isaac to make sure there isn't a reason the space is needed - thanks!

Reply:
The names are a bit of a historical artifact, since we've changed icon providers a couple times.